### DISCUSSION TITLE
This project needs a real description and simple explanation that didn't come from a jargon-generator.

### DISCUSSION THREAD

**Discussion by jessemcallisterperry** [IceWhaleTech/CasaOS#1531] [General]
An associate told me about your project. I looked at the this github page and what is the first description I get? A total garbage jargon string. 

"Connect with the community, establish autonomy, reduce the cost of SaaS, and MAXIMIZE the potential for a personalized copilot." 

Seriously? is the intention to totally obscure the purpose of this project?

**Comment by spuder** [comment]
Hi there, glad you are here. Thanks for opening an issue. Sorry the first line wasn't to your liking.

If you scroll down, there is much more information below including live demos, explanations, a dedicated webpage, a discord, multiple kickstarters, a feature list and screenshots. While not documented here there are also multiple YouTube videos containing walkthroughs. 

I'm confident that you will be able to find the information you are looking for there, if not please feel free to open a pull request and someone from IceWhale will review it.
